chapters/arrays.tex

```diff
@@ -1037,12 +1037,11 @@ \section{Scalar, Vector, Matrix, and Array Operator Functions}\label{scalar-vect
 
 \subsection{Equality and Assignment}\label{equality-and-assignment}
 
-Equality \lstinline!a = b! and assignment \lstinline!a := b! of scalars, vectors, matrices, and
-arrays is defined element-wise and require both objects to have the same
-number of dimensions and corresponding dimension sizes. The operands
-need to be type equivalent. This is legal for the simple types and all
-types satisfying the requirements for a record, and is in the latter
-case applied to each component-element of the records.
+Equality \lstinline!a = b! and assignment \lstinline!a := b! of scalars, vectors, matrices, and arrays is defined element-wise and require both objects to have the same number of dimensions and corresponding dimension sizes.
+See \cref{array-indexing} regarding assignments to array variables with repeated subscripts.
```

Comment:
I would write "vector of subscripts" instead of "repeated subscripts"; as the main point is about clearly defining `a[{i,j,k}]:={1,2,3}` as `a[i]:=1;a[j]:=2;a[k]:=3;` and the result for repeated subscripts just follow.

Reply:
Making myself a suggestion:
```suggestion
See \cref{array-indexing} regarding assignments to array variables with vector of subscripts.
```